dist/api_data.py

```diff
@@ -73,9 +73,9 @@ def __ge__(self, other):
         are the default, which ensures that once timestamps are used for a key, they are always
         used, and also that multiple updates to a key never use decreasing timestamps and
         \c never which enforces that timestamps are never used for a table. (The \c always,
-        \c key_consistent, \c mixed_mode and \c ordered choices should not be used, and are
-        retained for backward compatibility.)''',
-        choices=['always', 'key_consistent', 'mixed_mode', 'never', 'none', 'ordered']),
+        \c key_consistent, and \c ordered choices should not be used, and are retained for
+        backward compatibility.)''',
+        choices=['always', 'key_consistent', 'never', 'none', 'ordered']),
```

Comment:
Removing the option can cause backward compatibility problems when that option is used in older releases and later they try to import the table with that config. Refer to WT-8777 for a similar issue for `key_consistent` being removed and added back.

Reply:
Good to know! I put this back.